### PR TITLE
Actual lockers

### PR DESCRIPTION
**Pull request by PrPleGoo**
Adds storing entities into lockers the way we all know and love.
Relies on an implementation of ITileDefinition in https://github.com/space-wizards/space-station-14/pull/193 (just like origin/master)
#191

### CODE REVIEW

**Comment by PJB3005** [comment]
See I'm not sure we actually want SS13 style lockers.

The only real advantage they give IMO is making it easier to stick people in them, but otherwise they just make no sense.

**Comment by PrPleGoo** [comment]
SS13 style lockers also allow:
- onlookers to notice contents at a glance;
- any passerby to grab contents with a single click;
- onlookers to notice items being added/removed at a glance;
- a person to quickly dump the contents of an area into the locker by dropping, without refocusing the mouse onto the locker;
- a group of people to collaborate to throw items at an open locker;
If you want an inventory pane then just redo "alt click" to list items into an inventory screen, something that will need to be implemented anyway.

**Comment by PJB3005** [comment]
Reading your points I guess a middle ground where you can still use a saner menu than right click would be fine. Fair enough.

**Comment by PrPleGoo** [comment]
I'm not going to implement it for this PR but I will make an issue for it.

File: Content.Server/AI/AimShootLifeProcessor.cs [hunk 1]
```diff
@@ -119,7 +120,7 @@ private IEntity FindBestTarget()
 
                 // build the ray
                 var dir = entity.GetComponent<ITransformComponent>().WorldPosition - myTransform.WorldPosition;
-                var ray = new Ray(myTransform.WorldPosition, dir.Normalized);
+                var ray = new Ray(myTransform.WorldPosition, dir.Normalized, (int)CollisionGroup.Mob);
```
Comment (Acruid) [inline Content.Server/AI/AimShootLifeProcessor.cs]: This needs to collide with walls, the whole point of the ray is to check LOS with the player.
```suggestion
                var ray = new Ray(myTransform.WorldPosition, dir.Normalized, (int)(CollisionGroup.Mob | CollisionGroup.Grid));
```

Comment (PrPleGoo) [reply Content.Server/AI/AimShootLifeProcessor.cs]: Alright, thought this was a way to look for mobs.

**Pull request merged**